ui/templates/welcome.html

```diff
@@ -2,8 +2,17 @@
 {% block content %}
     <div class="section-header">
         <div class="row">
-            <div class="col-md-2 col-md-offset-5">{% if perms.learningresources.add_repository %}<a href="{% url 'create_repo' %}" role="button"
-                                                     class="btn btn-primary btn-large">Create repository</a>{% endif %}</div>
+            <div class="col-md-2 col-md-offset-5">
+                {% if perms.learningresources.add_repository %}
+                    <a href="{% url 'create_repo' %}"
+                       role="button"
+                       class="btn btn-primary btn-large">Create repository</a>
```

Comment:
Did you check the result on the browsers? I wrote the template in that weird way because I did not want to put spaces between the different tags.